install/Knative-custom-install.md

```diff
@@ -178,33 +180,33 @@ files from the Knative repositories:
-| [`monitoring-tracing-zipkin.yaml`][1.5]          | Installs only [Zipkin][2.3]. Requires installation of the ELK stack.\*                                |
-| [`monitoring-tracing-zipkin-in-mem.yaml`][1.6]   | Installs only [Zipkin in-memory][2.3]\*                                                               |
-| **knative/build**                                |                                                                                                       |
-| **[`release.yaml`][3.1]**                        | Installs the Build component.                                                                         |
-| **knative/eventing**                             |                                                                                                       |
-| **[`release.yaml`][4.1]**                        | Installs the Eventing component. Includes the in-memory channel provisioner.                          |
-| [`eventing.yaml`][4.2]                           | Installs the Eventing component. Does not include the in-memory channel provisioner.                  |
-| [`in-memory-channel.yaml`][4.3]                  | Installs only the in-memory channel provisioner.                                                      |
-| [`kafka.yaml`][4.4]                              | Installs only the Kafka channel provisioner.                                                          |
-| **knative/eventing-sources**                     |                                                                                                       |
-| **[`release.yaml`][5.1]**                        | Installs the following sources: [Kubernetes][6], [GitHub][6.1], [Container image][6.2], [CronJob][6.3]|
-| [`release-gcppubsub.yaml`][5.2]                  | Installs the following sources: [PubSub][6.4]                                                         |
-| [`message-dumper.yaml`][5.3]                     | Installs an Event logging service for debugging.                                                      |
+| Knative Install Filename                         | Notes                                                                                                 | Dependencies                                                      |
+| -------------------------------------------------| ----------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------- |
+| **knative/serving**                              |                                                                                                       |                                                                   |
+| **[`serving.yaml`][1.1]_†_**                     | Installs the  Serving component.                                                                      |                                                                   |
```

Comment:
```suggestion
| [`serving.yaml`][1.1]_†_                     | Installs the  Serving component.                                                                      |                                                                   |
```
remove the extra bold for all the † items in the tables and it works for me

Reply:
I ended up removing all the formatting from the new symbol; I don't think it needs it.